### PR TITLE
i18n(de): translate guides/pages

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `guides/pages.mdx` into German language.

### CODE REVIEW

**Comment by trueberryless** [comment]
Links fixed in #2458, #2470 and #2480

File: docs/src/content/docs/de/guides/pages.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Die Eigenschaft [`title`](/de/reference/frontmatter/#title-erforderlich) ist erforderlich und alle anderen Eigenschaften sind optional.
+
+Die folgenden Eigenschaften unterscheiden sich von Markdown frontmatter:
+
+- Die Eigenschaft [`slug`](/de/reference/frontmatter/#slug) wird nicht unterstützt und wird automatisch anhand der URL der benutzerdefinierten Seite gesetzt.
+- Die Option [`editUrl`](/de/reference/frontmatter/#editurl) erfordert eine URL, um einen Bearbeitungslink anzuzeigen.
+- Die Frontmatter-Eigenschaft [`sidebar`](/de/reference/frontmatter/#sidebar), mit der du festlegen kannst, wie die Seite in [autogenerierten Linkgruppen](/de/reference/configuration/#sidebar) erscheint, ist nicht verfügbar. Seiten, die die Komponente `<StarlightPage />` verwenden, sind nicht Teil einer Sammlung und können nicht zu einer automatisch erzeugten Seitenleistengruppe hinzugefügt werden.
+- Die Option [`draft`](/de/reference/frontmatter/#draft) zeigt nur einen [Hinweis](/de/reference/overrides/#draftcontentnotice) an, dass die Seite ein Entwurf ist, schließt sie aber nicht automatisch von der Produktion aus.
+
+##### `Sidebar`
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/pages.mdx:113]:
```suggestion
##### `sidebar`
```

This is a property name so should not be capitalized.

File: docs/src/content/docs/de/guides/pages.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+<StarlightPage
+	frontmatter={{ title: 'Orion' }}
+	sidebar={[
+		{ label: 'Home', link: '/' },
+		{
+			label: 'Constellations',
+			items: [
+				{ label: 'Andromeda', link: '/andromeda/' },
+				{ label: 'Orion', link: '/orion/' },
+				{ label: 'Ursa Minor', link: '/ursa-minor/', Abzeichen: 'Stub' },
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/pages.mdx:133]:
```suggestion
				{ label: 'Ursa Minor', link: '/ursa-minor/', badge: 'Platzhalter' },
```

The property name should not be translated and I think we used "Platzhalter" in https://starlight.astro.build/de/guides/sidebar/#abzeichenvarianten

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://contribute.docs.astro.build/reviewers/reviewing-translations/). Left a few comments/suggestions but overall it looks good to me and should be mergeable once links validation passes.

**Comment by trueberryless** [comment]
Thanks for the review, should be good to go!

**Review by HiDeoo — APPROVED** [review]
Thanks for the updates, we should be good to go now on this one too :star2:

**Pull request merged**